src/MSBuild/Resources/Strings.resx

```diff
@@ -1481,6 +1481,15 @@
       {4}: duration in seconds with 1 decimal point
     </comment>
   </data>
+  <data name="TerminalLoggerNotUsedDisabled" xml:space="preserve">
+    <value>TerminalLogger was not used because build is run in context of process (e.g. 'dotnet test') that requests direct access to stdout stream.</value>
```

Comment:
```suggestion
    <value>Terminal Logger was not used because build is run in context of a process (e.g. 'dotnet test') that requests direct access to stdout stream.</value>
```

(My best guess based on my limited English and the official name of the feature. @baronfel, can you please review these strings?)

Reply:
Adjusted.
But any other suggestions to formulations are definitely very welcome